src/server/app.js

```diff
@@ -31,35 +31,36 @@ const logs = require('./routes/logs');
 const obvius = require('./routes/obvius');
 const csv = require('./routes/csv');
 const conversionArray = require('./routes/conversionArray');
+const conversions = require("./routes/conversions");
 const units = require('./routes/units')
 
 // Limit the rate of overall requests to OED
 // Note that the rate limit may make the automatic test return the value of 429. In that case, the limiters below need to be increased.
 // Create a limit of 200 requests/5 seconds
 const generalLimiter = new rateLimit({
 	windowMs: 5 * 1000, // 5 seconds
-	max: 200 // 200 requests
+	max: 200, // 200 requests
```

Comment:
There should not be a final comma on the last time. Please remove here and below.